### PR TITLE
refactor(tools): add tools and images

### PR DESCRIPTION
**Pull request by testinaweb**

### CODE REVIEW

File: basics/tools.md [hunk 1]
```diff
@@ -7,60 +7,119 @@ Be very careful when people tell you to buy specific tools.
...
+## Desired
 
-Some kind of oven-proof plate on which you can put the dough or any other container in your oven.
-In case you have no pizza stone or dutch oven, it's advisable to cover this with parchment paper so your dough will not stick to it.
+### Baking Sheets/Paper
+You definitely want to have baking paper to avoid the dough to stick on the surface you will bake on. Instead of baking paper, you can buy a silicone baking mat which can be reuse oven and over, but it cannot be used inside a dutch oven.
 
-## Banneton (optional)
+### Kitchen Scale
+A kitchen scale is very useful even if it's no necessary, and if you are thinking to buy it, buy one very precise. Bread is an activity very close to chemistry, and most of the time you want to follow the quantities you find in the recepies.
```
Comment (jwaterworth) [inline basics/tools.md]: no necessary -> not necessary

File: basics/tools.md [hunk 2]
```diff
@@ -7,60 +7,119 @@ Be very careful when people tell you to buy specific tools.
...
 
-A banneton is used to let your dough rise one last time before baking it.
-Instead of using one bowl with parchment paper I prefer the banneton, as I don't need to buy parchment paper.
-Also to me it feels more natural to just let the dough rise in a wooden banneton.
-Additionally a banneton powered bread I believe looks visually a little more appealing.
-You have nice circles of flour on the top of your bread.
+### Dish cloth
```
Comment (jwaterworth) [inline basics/tools.md]: Is a tea towel also suitable instead?

File: basics/tools.md [hunk 3]
```diff
@@ -7,60 +7,119 @@ Be very careful when people tell you to buy specific tools.
...
+* [Dutch oven](#dutch-oven)
+* [Shower Cap](#shower-cap)
+* [Dough Scraper](#dough-scraper)
+* [Stand Mixer](#stand-mixer)
+* [Cooling Rack](#cooling-rack)
+
+## Required
+
+### Love
+Doing bread is a nobel activity, you need to put love in it. You take care of it and the result will change you perception of what bread is. This doesn't mean that you cannot do anything else at the same time. The bread preparation has to fit you life and not vice versa. Love is for sure something that you want to put in your bread.
```
Comment (jwaterworth) [inline basics/tools.md]: nobel or noble?

**Comment by jwaterworth** [comment]
A great read!!

**Comment by hendricius** [comment]
This pull request is amazing. Thank you very much!

**Comment by testinaweb** [comment]
Hi, why did you remove my changes?

**Pull request merged**